### DISCUSSION TITLE
Osquery index pattern

### DISCUSSION THREAD

**Discussion by m9o7h** [Security-Onion-Solutions/securityonion#6897] [Unsupported Versions]
Can i change the index pattern or the index template of the osquery agent (osquery package )?

if so can i build a new agent package with new configuration.
if not can i change the windows loging channel s or select all channels?

**Comment by defensivedepth** [comment]
What exactly are you trying to accomplish? (What is the end goal?)

**Reply by m9o7h** [reply]
I don't want to use the ":so-*" index pattern.
Instead, I'm creating new indexes with custom names to categorize the logs that the osquery collecting.
Actually I'm using the security onion as a log manager, when I use the custom index naming I can control the storage of the security onion.
Our environment has about 100 servers and 2000 windows users, so my goal is to collect all logs and manage the storage.

**Reply by defensivedepth** [reply]
Would be helpful if you could give some further detail.

You can use the  so- prefix, but name the suffix something different. You can then use custom Curator action files, etc.
